scss/_responsive.scss

```diff
@@ -0,0 +1,11 @@
+@media (max-width: $screen-mobile-max-width) {
```

Comment:
I suppose this is a matter of structuring the code; so we have two different methods:
1. Write the media queries "in-place", so we don't have a single `_responsive.scss` file, but all the responsive content will be written inside the appropriate files. This will result in a bit larger output file, since the `@media (max-width: $screen-mobile-max-width) {` will be duplicated many times (once per needed tag)
2. The way it is now, that we have a separate file for the responsive styles. This is somewhat how e.g. Bootstrap works.

I don't see so much bad in either style. If you have any preferences or better arguments than, I'd be happy to hear them :)